hipaa-npi-number-input.html

```diff
@@ -105,6 +109,8 @@
           this.set('invalid', this.invalid);
         } else {
           this.$.container.invalid = !valid;
```

Comment:
@anushagadepalli 
You need to set this value also to true.
I don't understand why it is being set to to `!valid`